airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -23,5 +23,10 @@ data:
   ab_internal:
     sl: 300
     ql: 400
+  releases:
+    breakingChanges:
+      4.0.0:
+        message: "Version 4.0.0 changes the cursors in most of the Stripe streams that support incremental sync mode. This is done to not only sync the data that was created since previous sync, but also the data that was modified. A schema refresh of all effected streams is required to use the new cursor format."
+        upgradeDeadline:
```

Comment:
Looks like you are still missing an `upgradeDeadline`, but :thumbsup: on the breaking change message